### DISCUSSION TITLE
flink doris connector能否实时读取doris数据

### DISCUSSION THREAD

**Discussion by Bin2020** [apache/doris#12227] [A - General / Q&A]
flinkCDC同步mysql到doris之后，想从doris里读取多张表到flink再进行join，弄成宽表重新插入doris。因为需要join的表在不同的数据库实例，flinkCDC只能同步一个数据库实例的多张表，所以不能在从mysql读取到flink时就进行join。目前尝试了从doris读取到flink是一次性的，有没有实时读取的功能呢。如果有其他方案也欢迎讨论。

**Comment by JNSimba** [comment]
doris source is currently a one-time read, you can use the jdbc connector to achieve lookup join

**Reply by Bin2020** [reply]
我试了一下，jdbc也是一次性读取的。难道是手动拼接sql的查询参数(where id=xxx)，每次都去查doris来实现join吗。。。
